workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -48,7 +48,7 @@ export const mapProcessInstanceToDetails = (
 
   return {
     id: instance.id,
-    name,
+    processName: name,
```

Comment:
I'm ok to have this as quick fix, but please open a jira issue to fix this in the backend. it shoudn't be FE resposnibility to properly map orchestrator objects into graphql ones.

Reply:
I think the best approach is for sonata, backend, and FE to all use the same propery names, otherwise the code becomes impossible to follow, and it's hard to understand what field names to use.
In this case the backend returns the correct field name processName, so the only fix required is for the UI to use it as is instead of converting it to "name".